--- origsrc/exempi-2.0.1/exempi/Makefile.am	2007-11-12 20:56:49.000000000 -0600
+++ src/exempi-2.0.1/exempi/Makefile.am	2008-05-16 01:09:29.078125000 -0500
@@ -54,4 +54,4 @@
 	$(top_builddir)/source/XMPFiles/libXMPFiles.la \
 	$(top_builddir)/third-party/MD5/libmd5.la \
 	-lexpat 
-libexempi_la_LDFLAGS = -version-info @EXEMPI_VERSION_INFO@
\ No newline at end of file
+libexempi_la_LDFLAGS = -version-info @EXEMPI_VERSION_INFO@ -no-undefined
--- origsrc/exempi-2.0.1/exempi/exempi.cpp	2008-04-23 23:52:28.000000000 -0500
+++ src/exempi-2.0.1/exempi/exempi.cpp	2008-05-16 01:51:29.953125000 -0500
@@ -51,7 +51,7 @@
 #include "XMP.incl_cpp"
 
 /* TODO make the thread local storage portable */
-static __thread int g_error = 0;
+static /* __thread */ int g_error = 0;
 
 static void set_error(int err)
 {
--- origsrc/exempi-2.0.1/source/XMPFiles/FormatSupport/Makefile.am	2007-12-12 20:22:26.000000000 -0600
+++ src/exempi-2.0.1/source/XMPFiles/FormatSupport/Makefile.am	2008-05-16 01:21:56.203125000 -0500
@@ -57,3 +57,5 @@
 	RIFF_Support.cpp     TIFF_Support.cpp \
 	PNG_Support.cpp   ReconcileLegacy.cpp  \
 	TIFF_FileWriter.cpp  XMPScanner.cpp  GIF_Support.cpp
+
+libformatsupport_la_LIBADD = -liconv
